.github/release_steps.md

```diff
@@ -1,42 +1,54 @@
 # LAMMPS Release Steps
 
-The following notes chronicle the current steps for preparing and publishing LAMMPS releases.  For
-definitions of LAMMPS versions and releases mean, please refer to [the corresponding section in the
-LAMMPS manual](https://docs.lammps.org/Manual_version.html).
+The following notes chronicle the current steps for preparing and
+publishing LAMMPS releases.  For definitions of LAMMPS versions and
+releases mean, please refer to [the corresponding section in the LAMMPS
```

Comment:
Spotted another minor error, I think this was either supposed to be "For definitions of LAMMPS versions and *what* releases mean" or preferably "For definitions of LAMMPS versions and releases."